include/eve/concept/scalar.hpp

```diff
@@ -28,7 +28,9 @@ concept plain_scalar_value = detail::one_of<T,
                                             std::uint8_t,
                                             std::uint16_t,
                                             std::uint32_t,
-                                            std::uint64_t>;
+                                            std::uint64_t,
+                                            std::size_t,
+                                            std::ptrdiff_t>;
```

Comment:
I think this is the wrong decision here, they are not plain_scalar. Didn't we discuss an alternative approach?

Reply:
Yes but it's elsewhere. SO I'll probably extract this exact fix and put it there.

Reply:
Is it staying? I'd like it to go away if possible.